.github/workflows/build_flang-driver.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches: [ release_90 ]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        target: [X86, AArch64, PowerPC]
+
+    steps:
+      - name: Check tools
+        run: |
+          git --version
+          cmake --version
+          make --version
+          gcc --version
+
+      - name: Download llvm
+        run: |
+          cd ../..
+          # First get the list of artifacts from the most recent run.
+          wget --output-document artifacts_llvm `curl -sL https://api.github.com/repos/flang-compiler/llvm/actions/workflows/build_llvm.yml/runs | jq -r '.workflow_runs[0].artifacts_url?'`
+          
+          echo "cat artifacts_llvm"
+          cat artifacts_llvm
+      
+          url=`jq -r '.artifacts[] | select(.name == "llvm_build_${{ matrix.target }}") | .archive_download_url' artifacts_llvm`
+          wget --output-document llvm_build.zip --header="Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" $url
+          
+      - name: Install llvm
+        run: |
+          cd ../..
+          # Check out the repo, so the sources are in place.
+          git clone --depth 1 --single-branch --branch release_90  https://github.com/flang-compiler/llvm.git
+          # Don't build - use the prebuilt build directory
+          unzip llvm_build.zip
```

Comment:
Wouldn't it make more sense to archive the LLVM sources within `llvm_build.zip`? Cloning `llvm.git` here might result in source files that are out of sync with the binaries in llvm_build.zip.

Reply:
Excellent point, @bryanpkc. I admit I didn't think about this.

Generally, the chances are low that we ever hit the "out of sync" situation, as there are just a few pushes per month to both `llvm` and `flang-driver` repos. 

Nevertheless, this change is worth considering. 
I haven't found any information on github limiting the space, zipped source code will take much less than 100 MB, while the current artifacts weigh more than 250 MB. The downloads may become faster and we'll decrease the script complexity. 

Quite a few gains and no losses, so I'll try this out on my fork and come back with an update to this PR if I don't come across any obstacles.

Reply:
This worked, @bryanpkc :smiley: 

See [llvm build](https://github.com/michalpasztamobica/llvm/actions/runs/337486934), [flang-driver build](https://github.com/michalpasztamobica/flang-driver/actions/runs/337592116) and [flang build](https://github.com/michalpasztamobica/flang/actions/runs/337689520) for results.

I placed changes in a separate commit to make the review easier. I'll squash and rename the project to `flang-compiler` once the review is approved.